sematic/api/endpoints/auth.py

```diff
@@ -28,10 +28,13 @@ def authenticate_endpoint() -> flask.Response:
     friction, we let users run locally without authentication.
     """
     providers = {}
-    authenticate = False
+    # Ideally we would do this normalization in user_settings.py,
+    # but it means we would need to type settings
+    authenticate = str_to_bool(
+        get_user_settings(SettingsVar.SEMATIC_AUTHENTICATE, False)
```

Comment:
Can we have a `get_bool_user_setting(...)` func that does the conversion instead?

Reply:
+1

Reply:
Done.